app/hotels/src/allHotels/AllHotels.js

```diff
@@ -13,41 +12,18 @@ import type { AllHotelsSearchQueryResponse } from './__generated__/AllHotelsSear
 import type { SearchParametersType } from './searchForm/SearchParametersType';
 
 type Props = {|
+  hotelsFilter: SearchParametersType,
   openSingleHotel: (id: string) => void,
+  onFilterChange: (filter: SearchParametersType) => void,
 |};
 
-type State = {|
-  search: SearchParametersType,
-|};
-
-export default class AllHotelsSearch extends React.Component<Props, State> {
-  state = {
-    search: {
-      latitude: 50.08,
-      longitude: 14.44,
-      checkin: null,
-      checkout: null,
-      roomsConfiguration: {
-        adultsCount: 1,
-        children: [],
-      },
-    },
-  };
-
+export default class AllHotelsSearch extends React.Component<Props, void> {
```

Comment:
`React.Component<Props>` (No need for the `, void>` part.)

Reply:
fixed